### PR TITLE
Set default value of extra_options to an empty string.

### PR DESCRIPTION
**Pull request by ghost**
I would like to write a test case for this, but I unfortunately do not know how -- maybe someone can teach me? Issue #20 details how the bug can be circumvented without the changes proposed in this PR.

### CODE REVIEW

**Comment by salimane** [comment]
@pong-takepart check the test case in https://github.com/meltwater/puppet-marathon/blob/master/spec/classes/init_spec.rb#L122 . It shows a case for defaults params and another for custom params

**Comment by pong-takepart** [comment]
I will clean up the commits. I just need your CI server to run tests for me first. Thanks.

File: spec/classes/init_spec.rb [hunk 1]
```diff
@@ -107,6 +107,41 @@
...
+      } }
+
+      it { should contain_file('marathon-conf').with({
+        :ensure => 'file',
+        :path   => '/etc/init.d/marathon'
+      }) }
+
+      context 'master and zk servers with defaults params' do
+        it 'should configure marathon with the correct arguments' do
+          should contain_file('marathon-conf').with_content(/args="--master zk:\/\/localhost:2181\/mesos --zk zk:\/\/localhost:2181\/marathon/ "/)
```
Comment (salimane) [inline spec/classes/init_spec.rb]: shouldn't this be 
``/args="--master zk:\/\/localhost:2181\/mesos --zk zk:\/\/localhost:2181\/marathon "/`` ?

Comment (pong-takepart) [inline spec/classes/init_spec.rb]: Yeah, I ran it locally and saw it. I changed it already. Looks like all checks passed.

**Comment by salimane** [comment]
@pong-takepart thank you very much for the PR. Release v0.4.3 has been published to https://forge.puppet.com/meltwater/marathon

**Comment by ghost** [comment]
@salimane I'm bragging to my coworkers now about how awesome I am. Thanks.

**Pull request merged**